### PR TITLE
Update COMMUNITY_FUND_POLICY.md - Add Age Requirement

### PR DESCRIPTION
**Pull request by bensternthal**
Added note that a person needs to be over 18.

### CODE REVIEW

**Review by ctcpip — APPROVED** [review]

**Comment by ljharb** [comment]
This is fine, but is this because it's the age of adulthood in the US, or is there a more concrete reason?

**Comment by bensternthal** [comment]
We discussed in the last CPC call, the primary reason is that this is the age of adulthood in the USA. We did not want to accept either the liability or overhead of implementing a program that could support funding the travel or attendance of minors.

**Comment by ljharb** [comment]
There are countries where that age is higher; Japan and NZ are 20, for example. I'm fine with 18 tho, for those stated reasons.

**Comment by ctcpip** [comment]
Maybe change it to 18 or the age of majority in the person's country of residence, whichever is higher. `Math.max()`

**Review by mcollina — APPROVED** [review]
lgtm

**Review by marco-ippolito — APPROVED** [review]

File: community-fund/COMMUNITY_FUND_POLICY.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The OpenJS Foundation’s community fund supports travel and attendance at event
 
 * Collaborators
 * Standards body participants
+* Over the age of 18
```
Comment (UlisesGascon) [inline community-fund/COMMUNITY_FUND_POLICY.md:9]:
```suggestion
* At least 18 years old and recognized as adults in the destination country, based on local laws (ref: [Age of Majority](https://en.wikipedia.org/wiki/Age_of_majority))
```

Maybe is safer to ensure that the requester is at least 18 (US Compliance) and also that can be considered "adult" in the destination country, that way we reduce the responsibilities a lot.

I can see some edge cases in the list (Canada +19, Puerto Rico +21, US (Nebraska) +19...)

**Review by ljharb — APPROVED** [review]

File: community-fund/COMMUNITY_FUND_POLICY.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ The OpenJS Foundation’s community fund supports travel and attendance at event
 
 * Collaborators
 * Standards body participants
+* At least 18 years old and recognized as adults in the destination country, based on local laws (ref: [Age of Majority](https://en.wikipedia.org/wiki/Age_of_majority))
```
Comment (ctcpip) [inline community-fund/COMMUNITY_FUND_POLICY.md:9]:
```suggestion
* At least 18 years old and recognized as adults in both the country of departure and the destination country, based on local laws (ref: [Age of Majority](https://en.wikipedia.org/wiki/Age_of_majority))
```

**Pull request merged**